modules/machine-health-checks-baremetal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * machine_management/deploying-machine-health-checks.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-health-checks-baremetal_{context}"]
+= MachineHealthChecks on Bare Metal
+
+Machine deletion on bare metal cluster triggers reprovisioning of a bare metal host.
+Usually bare metal reprovisioning is a lengthy process, in which the cluster 
+is lacking compute power, and some applications are down.
```

Comment:
Not a guarantee that losing one host would take down applications so maybe reword as below, WDYT?
```suggestion
is missing compute resources and applications may be interrupted.
```

Reply:
s/may/might